src/chain/chain.rs

```diff
@@ -1571,6 +1576,55 @@ fn compute_relocation_destination(
         .unwrap_or_else(|| relocation::compute_destination(relocated_name, trigger_name))
 }
 
+#[derive(Clone)]
+/// The secret share of the section key.
+pub struct SectionKeyShare {
```

Comment:
Nitpick: is there a convention for doc comment vs `#[]` order? I see it's mixed here...

Reply:
I personally put doc comments first. Not sure that is an established convention though.